diff -ur bsd-games-2.15/monop.orig/Makefrag bsd-games-2.15/monop/Makefrag
--- bsd-games-2.15/monop.orig/Makefrag	Sun Sep 26 13:37:19 1999
+++ bsd-games-2.15/monop/Makefrag	Wed Sep 22 13:15:56 2004
@@ -35,7 +35,8 @@
 	$< monop/cards.inp $@
 
 monop_install:	monop_all
-	$(INSTALL_BINARY) monop/monop $(INSTALL_PREFIX)$(GAMESDIR)/monop
-	$(HIDE_GAME) monop
-	$(INSTALL_MANUAL) monop/monop.6
+	$(INSTALL_BINARY) monop/monop $(INSTALL_PREFIX)$(GAMESDIR)/monopoly-game
+	$(HIDE_GAME) monopoly-game
+	mv monop/monop.6 monop/monopoly-game.6
+	$(INSTALL_MANUAL) monop/monopoly-game.6
 	$(INSTALL_DATA) monop/cards.pck $(INSTALL_PREFIX)$(MONOP_CARDSFILE)
diff -ur bsd-games-2.15/monop.orig/monop.6.in bsd-games-2.15/monop/monop.6.in
--- bsd-games-2.15/monop.orig/monop.6.in	Wed Dec 17 03:46:49 2003
+++ bsd-games-2.15/monop/monop.6.in	Wed Sep 22 13:04:41 2004
@@ -30,10 +30,10 @@
 .\"	@(#)monop.6	6.5 (Berkeley) 3/25/93
 .\"
 .Dd March 25, 1993
-.Dt MONOP 6
+.Dt MONOPONY-GAME 6
 .Os
 .Sh NAME
-.Nm monop
+.Nm monopoly-game
 .Nd Monopoly game
 .Sh SYNOPSIS
 .Nm
